src/main/java/org/folio/reader/AbstractEntityReader.java

```diff
@@ -1,15 +1,19 @@
 package org.folio.reader;
 
+import lombok.extern.log4j.Log4j2;
 import org.folio.processor.rule.DataSource;
 import org.folio.processor.rule.Rule;
 import org.folio.reader.values.MissingValue;
 import org.folio.reader.values.RuleValue;
 
 @SuppressWarnings("java:S3740")
+@Log4j2
 public abstract class AbstractEntityReader implements EntityReader {
 
   @Override
   public RuleValue read(Rule rule) {
+    log.info("read:: parameters rule: {}", rule);
```

Comment:
Rule is object. I'm not sure that it has custom toString() implementation to show needed information.

Reply:
@khandramai 
done

![image](https://github.com/folio-org/generate-marc-utils/assets/141402754/e1cdaf45-be02-4742-88df-0c6e21519b45)